mapswipe_workers/mapswipe_workers/utils/api_calls.py

```diff
@@ -27,33 +29,76 @@ def geojsonToFeatureCollection(geojson: dict) -> dict:
     return geojson
 
 
-def query_osm(changeset_id: int):
+osm_results = {}
```

Comment:
I'm not sure if there is another option to setting this here. From my point of view it might make more sense to define this in the query_osm function and then return `osm_results` at the end of the function.